src/main/java/it/gov/pagopa/cgn/portal/repository/BucketCodeLoadRepository.java

```diff
@@ -3,7 +3,11 @@
 import it.gov.pagopa.cgn.portal.model.BucketCodeLoadEntity;
 import org.springframework.data.jpa.repository.JpaRepository;
 
+import java.util.List;
+
 public interface BucketCodeLoadRepository extends JpaRepository<BucketCodeLoadEntity, Long> {
 
     BucketCodeLoadEntity findByDiscountIdAndUid(Long discountId, String uid);
+
+    List<BucketCodeLoadEntity> findByDiscountId(Long discountId);
```

Comment:
findBy and findAllBy have exactly the same behaviour, but you are right, it's better to use findAllBy because we used it also for other methods, and we should be coherent.

Reply:
fixed https://github.com/pagopa/cgn-onboarding-portal-backend/pull/107/commits/9e9779ea784ca535665776493e2eb2b14a7bb3b3